### PR TITLE
Stand Arrow Tweaks and Fixes

### PR DESCRIPTION
**Pull request by yobob591**
Tweaks arrow kill chance down a bit in exchange for husking to stop defib cheese, also fixes shit with the chaos guardian admin logs and not broadcasting stab messages to nearby players, meaning you now know when someones gonna stab you.

### CODE REVIEW

**Comment by banthebantz** [comment]
can you give the stand limited uses please so the entire crew doesn't get stands every round.

**Comment by yobob591** [comment]
no the reason it exists is infinite uses, what i can do is make it harder to get or more punishing

**Comment by ArcLumin** [comment]
Gib or husk is fine by me

File: code/modules/mob/living/simple_animal/guardian/guardian.dm [hunk 1]
```diff
@@ -625,8 +627,8 @@
 		used = TRUE
 	if(killchance == TRUE)
 		if(prob(percentchance))
-			user.visible_message("You didnt have enough fighting spirit!")
-			user.setToxLoss(100000) //Husks them to stop clone cheeze (not anymore now that its on mining)
+			user << "You didnt have enough fighting spirit!"
+			user.adjustFireLoss(100000) //Husks them to stop clone cheeze (not anymore now that its on mining)
```
Comment (JohnGinnane) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: Does this much fire damage husk?

**Comment by JohnGinnane** [comment]
Code looks good

File: code/modules/mob/living/simple_animal/guardian/guardian.dm (hunk 1 above)
Comment (ArcLumin) [reply code/modules/mob/living/simple_animal/guardian/guardian.dm]: I MEAN

**Comment by KittyKathy** [comment]
Is the Standroom supposed to spawn every round?

**Comment by ArcLumin** [comment]
Yes, all mining rooms were boosted from 80% spawn to 100%

**Comment by KittyKathy** [comment]
Well the stand room sometimes does not spawn on the mining map at all so some rounds it is impossible to get the stand arrow.

**Comment by ArcLumin** [comment]
So far, admins have not reported it not appearing, nor has it not appeared during testing. I do know artifact room logging for room teleportation is hella bugged, so trying to find it without admin powers can be a headache. However, it is possible that some currently unknown bug causes it to not properly spawn

**Comment by MethMedic** [comment]
Ive seen it be found several rounds in a row nearly at round start. No problems so far.

**Comment by Swagile** [comment]
honestly great feature, just gotta make it so you can't defib. you can get around this by cloning, but that takes a lot of time and effort to do that most people won't put into just to get one stand at round end

**Comment by KittyKathy** [comment]
It was confirmed to not be spawned or found in the asteroid one round i was a observer and looked all over then asked a admin BardOfRage if they could help me locate it and they also could not find it maybe someone just blew the room up or something? You can find them as a ghost by teleporting to the Artifact Rooms and checking them

**Comment by yobob591** [comment]
I actually buffed the spawnrate AGAIN in this PR simply because of that, hopefully that fixes it otherwise its a bug with templates and you can blame /tg/

**Comment by Sarcalogo** [comment]
Here's what I would do to abuse it. Sign up as Captain, get myself prescanned in the cloner. Go on out to mining and find it. Go to genetics. Use it on myself. If I died, have someone clone me. If not, I now have a free holoparasite.

What's to stop powergamers from doing that?

**Comment by Pyko1** [comment]
With properly updated cloner dying is a minor inconvience, especially when you are already loaded up and set to auto clone.

And the science should be easily able to do that while you are digging away.

**Comment by Pyko1** [comment]
Also mining secret generation has nothing to do with templates as far as i know.

Also I wonder if make_mining_asteroid_secret() will do once all themes are depleted from the list.

File: code/game/asteroid.dm [hunk 2]
```diff
@@ -1,6 +1,6 @@
 var/global/list/possiblethemes = list("organharvest","cult","wizden","cavein","xenoden","hitech","speakeasy","plantlab", "standroom")
 
-var/global/max_secret_rooms = 10
+var/global/max_secret_rooms = 12
```
Comment (ghost) [inline code/game/asteroid.dm:3]: What is this doing?

File: code/modules/mob/living/simple_animal/guardian/guardian.dm (hunk 1 above)
Comment (ghost) [reply code/modules/mob/living/simple_animal/guardian/guardian.dm]: "didnt"
FIX IT

File: code/game/asteroid.dm (hunk 2 above)
Comment (KayeArray) [reply code/game/asteroid.dm:3]: Oh shit son

**Comment by yobob591** [comment]
Well you can already cheese it super easy with defibs at round start with only one other person, the thing with captain and science would at least take longer or require science to actually work with the captain, which means you deserve it for that much work

**Comment by McBawbaggings** [comment]
I think think this should have been part of a meteorite event so that they are rare but make them more useful.

**Comment by yobob591** [comment]
Yeah I'd love to make it an event, maybe I'll do that next but for now I say we merge this to make it less broken, cause right now you can stab people without them knowing until they are already dead or have a stand and chaos holoparas have no admin logs for setting people on fire so this is pretty priority

**Comment by SpankMasterson** [comment]
kk +1

**Comment by yobob591** [comment]
@McBawbaggings FIXED ' REEEEEEEE

**Comment by KittyKathy** [comment]
You should probably fix that you can spam the arrow as in use it on yourself or someone several times before it does anything because this can lead to you rolling death on one and get a stand on another.
If that happens you die then get a stand and get gibed or get a stand then die and get gibed.
Should make it so you can't use if you are already using it on that same mob so you can't stab them like 3 times or more before the first first progress bar finishes.

Like i think you can legit spam it on someone or at least yourself right now to get them a stand and death causing them to gib.

**Comment by yobob591** [comment]
you can spam yourself yes, in fact i think im just gonna remove the use key and make you click yourself

**Comment by KittyKathy** [comment]
Wait if you can spam yourself and you get lucky and never roll death does that mean you can get like more than one stand?

**Comment by yobob591** [comment]
yeah I saw that happen recently, some dude spammed it and got 3 stands it was pretty greato daze

**Comment by McBawbaggings** [comment]
WHY CAN YOU STACK STANDS?

**Comment by yobob591** [comment]
you can give stands stands due to the killchance, but you having more than one stand is a bug

**Comment by ArcLumin** [comment]
Just put a check of "If it's being used don't let it be used again"

**Comment by yobob591** [comment]
removes the use key feature meaning you now have a 3 second delay on yourself, should stop you from glitching it to give yourself multiple standos

**Comment by yobob591** [comment]
also will add in use check

**Comment by Swagile** [comment]
GM + 1 if this gibs and honestly it still takes longer with a updated cloner than to just defib a dude, give him some meds / put in cryo then do it again.

**Comment by yobob591** [comment]
it husks, which is pretty much the same as gibbing since you can just scan yourself first either way

**Comment by yobob591** [comment]
the idea is just to stop defib cheese

File: code/game/asteroid.dm (hunk 2 above)
Comment (ArcLumin) [reply code/game/asteroid.dm:3]: It shouldn't do anything. Once it's used it's removed from attempting to spawn a room, so after all 9 themes are used no more will spawn.

Comment (ArcLumin) [reply code/game/asteroid.dm:3]: I guess it's just for "if someone else wants to make a room, this is here so it's still 100% spawn."

**Comment by yobob591** [comment]
uh can this be merged or does it need head input??

**Comment by McBawbaggings** [comment]
Does this prevent multiple stands on the one user?

**Comment by yobob591** [comment]
yes

**Comment by JohnGinnane** [comment]
Did you test this locally?

**Comment by yobob591** [comment]
yes

**Comment by yobob591** [comment]
it works, it checks if you are currently stabbing someone or yourself and stops you from doing it again

**Comment by yobob591** [comment]
well until the stab is finished

**Comment by yobob591** [comment]
meaning no mo stando spamo

**Comment by McBawbaggings** [comment]
Okay +1

**Pull request merged**